src/resources/Organizations/OrganizationInterfaces.ts

```diff
@@ -16,22 +16,31 @@ export interface OrganizationsStatusModel {
     supportActivated: boolean;
 }
 
-export interface OrganizationModel {
+export type OrganizationModel = {
     id: string;
     displayName: string;
     createdDate: number;
-    license: LicenseModel;
-    organizationStatusModel: OrganizationsStatusModel;
     publicContentOnly: boolean;
     type: string;
     owner: {
         email: string;
     };
     readOnly: boolean;
+} & AdditionalOrganizationFieldsModel;
+
+export interface AdditionalOrganizationFieldsModel {
+    license?: LicenseModel;
+    organizationStatusModel?: OrganizationsStatusModel;
```

Comment:
why extract those out of the OrganizationModel ?

Reply:
Originally wanted to extract the keys from the interface, but I noticed that for `organizationStatusModel`, we needed to pass the `status` additional field 🙄 

I like that they are tagged as "Additional" though, it implies that they are not returned by default.

But yeah, I guess I could put them back